pkgs/applications/networking/synology/drive.nix

```
@@ -0,0 +1,91 @@
+{ pkgs }:
+with pkgs;
+let
```

```suggestion
{ ... }:

let
```

Please replace ... with the individual packages.